<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WKWZTPqvmdckJHX" failures="2" tests="11" name="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" time="157.705" errors="0" timestamp="2023-01-24T18:31:46 IST" skipped="0">
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="cookieButton" time="19.033"/>
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyBackToTheArticleList" time="12.225"/>
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyIfYourTeethLinkOne" time="14.591">
    <failure message="expected [amp/my-teeth-hurt-pain-remedies.html] but found [https://www.sensodyne.com/en-us/amp/my-teeth-hurt-pain-remedies.html]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [amp/my-teeth-hurt-pain-remedies.html] but found [https://www.sensodyne.com/en-us/amp/my-teeth-hurt-pain-remedies.html]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:752)
at com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest.verifyIfYourTeethLinkOne(SensodyneWhyTeethHurtColdOutsideTest.java:67)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyIfYourTeethLinkOne -->
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyIfYourTeethLinkThree" time="12.728"/>
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyLivingWithToothSensitivityLinkTwo" time="16.729"/>
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyLivingWithToothSensitivityLinkThree" time="13.991"/>
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyHowToFixToothSensitivityLink" time="17.644">
    <failure message="expected [oral-health-tips/science-of-sensodyne/engineered-for-speed/] but found [https://www.sensodyne.com/en-us/oral-health-tips/science-of-sensodyne/engineered-for-speed/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [oral-health-tips/science-of-sensodyne/engineered-for-speed/] but found [https://www.sensodyne.com/en-us/oral-health-tips/science-of-sensodyne/engineered-for-speed/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.baseClass.BaseClass.compareTwoText(BaseClass.java:752)
at com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest.verifyHowToFixToothSensitivityLink(SensodyneWhyTeethHurtColdOutsideTest.java:140)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyHowToFixToothSensitivityLink -->
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyRelatedArticleOne" time="12.901"/>
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyRelatedArticleTwo" time="12.986"/>
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyRelatedArticleThree" time="11.470"/>
  <system-out/>
  <testcase classname="com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest" name="verifyRelatedArticleFour" time="13.407"/>
  <system-out/>
</testsuite> <!-- com.qa.sensodyne.us.test.SensodyneWhyTeethHurtColdOutsideTest -->
